athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchMetadataHandler.java

```diff
@@ -117,6 +136,7 @@ protected ElasticsearchMetadataHandler(AWSGlue awsGlue,
         this.domainMap = this.domainMapProvider.getDomainMap(null);
         this.clientFactory = clientFactory;
         this.glueTypeMapper = new ElasticsearchGlueTypeMapper();
+        this.queryTimeout = new Long(30);
```

Comment:
allow this to be set via constructor. Also, why is this a Long (boxed type) vs. long (primitive type)?

Reply:
changed to final long and passed via constructor.